src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -2,6 +2,7 @@
 
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Utilities;
```

Comment:
Why?

Reply:
Woops, I was moving that method around, will remove